### ISSUE TITLE
Bug: SCMLayer.GetLatestReleaseVersion nil pointer

### ISSUE THREAD

**Issue by eksrha** [fullstack-devops/awesome-ci#139]
**Describe the bug**
Nil pointer

**Your environment/setup**
- [x] GitHub Actions
- [ ] GitLab CI
- [ ] Jenkins CI

How is this tool installed
- Container

Are all environment variables set? See README.md

- yes
- no

**To Reproduce**
Paste your executed command here:

Paste your Pipeline log (this is optional, make shure no secrets are visable):

```shell
Run awesome-ci -v pr info -n 50
time="2023-03-02T12:32:12Z" level=info msg="detected ces type: github_runner"
time="2023-03-02T12:32:12Z" level=info msg="evaluated pull request number 50"
time="2023-03-02T12:32:12Z" level=info msg="found pull request 'Bump commons-text from 1.9 to 1.10.0' with given number 50"
time="2023-03-02T12:32:12Z" level=trace msg="got PR information from scm-portal-layer"
time="2023-03-02T12:32:12Z" level=trace msg="completed PR standard information to:  {50 em*** pm*** bugfix   f5a776e56f2d41e79c5208fd77150b92425e1559 f5a776e5 dependabot/maven/org.apache.commons-commons-text-1.10.0 3167bad4a1c7cded99e30ff9d0cb0c4ffdc12e7d}"
time="2023-03-02T12:32:12Z" level=trace msg="comment help instructions to issue"
time="2023-03-02T12:32:13Z" level=trace msg="commented help instructions to issue"
time="2023-03-02T12:32:13Z" level=trace msg="read comments from pr/mr and looking for overrides"
time="2023-03-02T12:32:13Z" level=trace msg="read comments from pr/mr complete conclusions (if nil no override), version: <nil>, patchLevel: <nil>"
time="2023-03-02T12:32:13Z" level=trace msg="get latest release, if any"
time="2023-03-02T12:32:13Z" level=trace msg="querying release page 1"
time="2023-03-02T12:32:13Z" level=trace msg="found 38 releases at page 1 begin with mapping..."
time="2023-03-02T12:32:13Z" level=trace msg="####### next page: 0"
time="2023-03-02T12:32:13Z" level=trace msg="ended with paging through releases"
time="2023-03-02T12:32:13Z" level=trace msg="open local git repository..."
time="2023-03-02T12:32:13Z" level=trace msg="...opened local git repository"
time="2023-03-02T12:32:13Z" level=trace msg="[0.0.1 7.0.0 10.5.1 10.5.2 10.5.3 10.5.4 10.5.5 10.5.6 10.5.7 10.6.0 10.7.0 10.7.1 10.7.2 10.7.3 10.7.4 10.7.5 10.7.6 10.8.0 10.8.1 10.8.2 10.8.3 10.9.0 10.9.1 10.10.0 10.11.0 10.11.1 10.12.0 10.12.1 10.13.0 10.14.0 10.15.0 10.15.1 10.16.0 10.17.0 10.18.0 10.18.1 10.19.0 10.20.0]"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xabb3fc]

goroutine 1 [running]:
github.com/fullstack-devops/awesome-ci/internal/app/awesome-ci/scm-portal.SCMLayer.GetLatestReleaseVersion({{0xc6f480, 0xc00020ab70}, {{0xcf5149, 0xd}, {0xc0000300eb, 0x5a}, 0xc000129be0}})
	/home/runner/work/awesome-ci/awesome-ci/internal/app/awesome-ci/scm-portal/release.go:24 +0x15c
github.com/fullstack-devops/awesome-ci/internal/app/awesome-ci/scm-portal.SCMLayer.GetPrInfos({{0xc6f480, 0xc00020ab70}, {{0xcf5149, 0xd}, {0xc0000300eb, 0x5a}, 0xc000129be0}}, 0x0?, {0x0, 0x0})
```

**Expected behavior**
no nil pointer

**Additional context**

**Comment by eksrha** [comment]
The cause was an outdated draft publication in releases, awesome-ci now ignores this and informs about it in the logs.

**Comment by eksrha** [comment]
resolved by #140
